.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "wcwave_adaptors"]
-	path = wcwave_adaptors
-	url = https://github.com/tri-state-epscor/wcwave_adaptors.git
```

Comment:
Ok, that is great. 

You didn't make a mistake.  I should have communicated how wcwave_adaptors as a submodule should be handled. I'm pretty sure I can merge your changes without affecting wcwave_adaptors. I am only asking about this so that I understand the difficulties we've encountered and can communicate better in the future.